ribbon/Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+web3 = "*"
+eth-tester = "*"
+eth-keys = "*"
+python-dotenv = "*"
+pyjwt = "*"
+cryptography = "*"
+eth-utils = "*"
+
+[dev-packages]
+ipykernel = "*"
+
+[requires]
+python_version = "3.9"
```

Comment:
Should we use `"3"`? 
To allow people using 3.8 and 3.10 as well, I have 3.10 on my machine.

Reply:
Sure, we can go with 3, was following the convention of the api repo.

Reply:
Thanks, let's keep it open for now and see later 🙏🏽